functions/index.js

```diff
@@ -0,0 +1,8 @@
+var functions = require('firebase-functions');
+
+// // Create and Deploy Your First Cloud Functions
```

Comment:
This is good for future extendability, this could be really fucking big if we could get some public contracts going with a public caching layer that you could interact with via cloud functions.